Src/MultiSafepay/Model/GatewayInfo.cs

```diff
@@ -107,14 +107,25 @@ public static GatewayInfo Einvoice(DateTime? birthday, string bankAccount, strin
             };
         }
 
-        public static GatewayInfo Klarna(DateTime birthday, string gender, string phone, string email)
+        public static GatewayInfo Klarna(DateTime? birthday, string gender, string phone, string email)
```

Comment:
Made nullable because in case of a redirect order the required values will be prompted on the redirect page.